javascript/javascript3/week1/weatherApi.js

```diff
@@ -0,0 +1,97 @@
+//Weather API------------------------------------------------------------------
+//get weather for your choice of city------------------------------------------
+
+const url = c20bdbd9cdf94e9fb3889776572cba5a;
```

Comment:
This here should be a string:

```suggestion
const url = 'c20bdbd9cdf94e9fb3889776572cba5a';
```

I would also say that calling this value `url` is a bit incorrect. Calling it `apiKey` or `appId` would be more accurate.